### PR TITLE
Add sample for `--ignore-pattern` to use with mutliple pattern

### PR DESCRIPTION
**Pull request by marianfoo**
**Improve the explanation of how multiple patterns can be used in the --ignore-pattern option, as it is not obvious**

**Thank you for your contribution!** 🙌

To get it merged faster, kindly review the checklist below:

## Pull Request Checklist
- [ ] Reviewed the [Contributing Guidelines](https://github.com/SAP/ui5-linter/blob/main/CONTRIBUTING.md#-contributing-code)
    + Especially the [How to Contribute](https://github.com/SAP/ui5-linter/blob/main/CONTRIBUTING.md#how-to-contribute) section 
- [ ] [No merge commits](https://github.com/SAP/ui5-linter/blob/main/docs/Guidelines.md#no-merge-commits)
- [ ] [Correct commit message style](https://github.com/SAP/ui5-linter/blob/main/docs/Guidelines.md#commit-message-style)

### CODE REVIEW

**Comment by flovogt** [comment]
@marianfoo thanks a lot for your contribution.

**Comment by maxreichmann** [comment]
Helpful contribution. LGTM

File: README.md [hunk 1]
```diff
@@ -119,7 +119,7 @@ Pattern/files that will be ignored during linting. Can also be defined in `ui5li
 
 **Example:**
 ```sh
-ui5lint --ignore-pattern "webapp/thirdparty/**"
+ui5lint --ignore-pattern "webapp/thirdparty/**" "webapp/test/**"
```
Comment (d3xter666) [inline README.md:122]: This one is also valid:
`ui5lint --ignore-pattern "webapp/thirdparty/**" --ignore-pattern "webapp/test/**"

Comment (marianfoo) [reply README.md:122]: Does it make sense to have this in the readme as well?

Comment (matz3) [reply README.md:122]: I think adding another `--ignore-pattern` is only useful when generically adding options to a commend, e.g. when calling the CLI programatically. When using the option directly, it's easier to do it as suggested in this PR, so I think we should only document that one.

Side-note: I'd suggest to use `"webapp/test/e2e/**"` as an example pattern, as test should be linted, but there may be end-to-end tests e.g. in Node.js (wdio).

Comment (marianfoo) [reply README.md]: > Side-note: I'd suggest to use "webapp/test/e2e/**"

changed it

**Review by matz3 — COMMENTED** [review]
LGTM

**Review by d3xter666 — APPROVED** [review]
LGTM

**Comment by marianfoo** [comment]
Thanks!

**Pull request merged**